integration/test-integration/src/test/resources/apim/config.toml

```diff
@@ -139,6 +139,13 @@ disableSslVerification = false
     # Certificate Filepath within enforcer
     certificateFilePath = "/home/wso2/security/truststore/mg.pem"
 
+# Issuer 3
+[[security.enforcer.tokenService]]
+    name="APIM_Publisher"
```

Comment:
This does not match the constant used

Reply:
Yes. Will fix. mistakenly added to this apim config toml

Reply:
Fixed with https://github.com/wso2/product-microgateway/pull/1905/commits/bd3fa32ad0a3c8ffd40ce8bd83e833ba79a5adcc